src/lib/github/prComments.ts

```diff
@@ -145,3 +150,82 @@ export const createPrComment = async (
   return ghResponse.data as ghPrCommentsStatus
 };
 
+export const deletePrComment = async (
+  ghDetails: ghDetails
+) : Promise<void> => {
+
+  const baseUrl = process.env.GH_API || 'https://api.github.com';
+  const commentUrl = `/repos/${ghDetails.orgName}/${ghDetails.repoName}/issues/${ghDetails.prNumber}/comments`;
+  let ghResponse;
+  const commentUrlList: string[] = []
+
+  const requestHeaders: Record<string, any> = {
+    'Content-Type': 'application/json',
+    Authorization: `token ${ghDetails.token}`,
+  };
+
+  const ghClient = axios.create({
+    baseURL: baseUrl,
+    responseType: 'json',
+    headers: { ...requestHeaders },
+  });
+
+  ghResponse = await ghClient.get(
+    commentUrl
+  );
+
+  // Need to get only the snyk comments
+  if ((ghResponse.data.length != 0)) {
+    ghResponse.data.map((comments: any) => {
+      if (comments.body.includes('******* Vulnerabilities report for commit')) 
+      {
+        commentUrlList.push(comments.url)
+      }
+    }) 
+  }
+
+  // Delete them
+  commentUrlList.forEach(async url => {
+    debug(`Delete comment at url: ${url}`)
+    ghResponse = await ghClient.delete(
+      url);
+  });
+
+  return 
+}
+
+export const prCommentExist = async (
```

Comment:
This function feels unnecessary. If there are pr comments that exist we'll delete them anyways, so you can just call delete which already pulls the comments to know which one(s) to delete. No need to do this work twice.